msvc/projects/vc2017/jemalloc/jemalloc.vcxproj.filters

```diff
@@ -133,6 +133,9 @@
     <ClCompile Include="..\..\..\..\src\prof_sys.c">
       <Filter>Source Files</Filter>
     </ClCompile>
+    <ClCompile Include="..\..\..\..\src\prof_threshold.c">
+      <Filter>Source Files</Filter>
+    </ClCompile>
```

Comment:
Similar to here, two more files also need updating:
msvc/projects/vc2019/jemalloc/jemalloc.vcxproj.filters
msvc/projects/vc2022/jemalloc/jemalloc.vcxproj.filters